prisma/schema/app.prisma

```diff
@@ -90,28 +90,33 @@ model CoreService {
 }
 
 model Booking {
-    id              String        @id @default(auto()) @map("_id") @db.ObjectId
-    date            String
-    startTime       String
-    endTime         String
-    notes           String?
-    duration        Int
-    price           Float
-    rooms           Float?
-    squareFootage   Float?
-    status          BookingStatus @default(PENDING)
-
-    serviceId       String 
-    service         CoreService   @relation(fields: [serviceId], references: [id], onDelete: Cascade)
-
-    customerId String   @db.ObjectId
-    customer   Customer @relation(fields: [customerId], references: [id])
-
-    businessId String   @db.ObjectId
-    business   Business @relation(fields: [businessId], references: [id])
-
-    createdAt DateTime @default(now())
-    updatedAt DateTime @updatedAt
+    id                  String        @id @default(auto()) @map("_id") @db.ObjectId
+    date                String
+    startTime           String
+    endTime             String
+    notes               String?
+    duration            Int
+    price               Float
+    rooms               Float?
+    squareFootage       Float?
+    status              BookingStatus @default(PENDING)
+    paymentIntentId     String?
+    completedAt         DateTime?
+    originalBookingId   String?
+    receiptUrl          String?
+    checkoutUrl         String?
+
+    serviceId           String 
+    service             CoreService   @relation(fields: [serviceId], references: [id], onDelete: Cascade)
+
+    customerId          String   @db.ObjectId
+    customer            Customer @relation(fields: [customerId], references: [id])
+
+    businessId          String   @db.ObjectId
+    business            Business @relation(fields: [businessId], references: [id])
+
+    createdAt           DateTime @default(now())
+    updatedAt           DateTime @updatedAt
 
```

Comment:
Has the whole team including @rai-dorzback @qhuboo signed off on these schema changes?